internal/mode/static/manager.go

```diff
@@ -134,7 +137,7 @@ func StartManager(cfg config.Config) error {
 	}
 
```

Comment:
Let's create `processHandler` here right before it's used.

Reply:
sounds amazing